packages/cli/src/WorkflowExecuteAdditionalData.ts

```diff
@@ -953,21 +955,23 @@ async function executeWorkflow(
 
 	void internalHooks.onWorkflowPostExecute(executionId, workflowData, data, additionalData.userId);
 
-	if (data.finished === true) {
+	// subworkflow either finished, or is in status waiting due to a wait node, both cases are considered successes here
+	if (data.finished === true || data.status === 'waiting') {
```

Comment:
Would it make sense to change the check to `data.status !== 'error'`?

Reply:
It would not be just error I guess, it would have to be canceled, crashed, error, failed, unknown, warning, new - so I think this way around is easier